app/controllers/provider_foreman_controller.rb

```diff
@@ -527,17 +532,20 @@ def provider_node(id, model)
     else
       case @record.type
       when "ManageIQ::Providers::Foreman::ConfigurationManager"
-        options = {:model => "ConfigurationProfile", :match_via_descendants => ConfiguredSystem}
-        options[:where_clause] = ["manager_id IN (?)", provider.id]
+        options = {:model => "ConfigurationProfile", :match_via_descendants => ConfiguredSystem,:where_clause => ["manager_id IN (?)", provider.id]}
         @no_checkboxes = true
         process_show_list(options)
         add_unassigned_configuration_profile_record(provider.id)
-        record_model = ui_lookup(:model => model ? model : TreeBuilder.get_model_for_prefix(@nodetype))
+        record_model = ui_lookup(:model => model_to_name(model || TreeBuilder.get_model_for_prefix(@nodetype)))
         @right_cell_text = _("%{model} \"%{name}\"") % {:name  => provider.name,
-                                                        :model => "#{ui_lookup(:tables => "configuration_profile")} under #{record_model}"}
+                                                        :model => "#{ui_lookup(:tables => "configuration_profile")} under #{record_model} Provider"}
       when "ManageIQ::Providers::AnsibleTower::ConfigurationManager"
-        @right_cell_text =
-          _("%{model} \"%{name}\"") % {:name => provider.name, :model => "#{record_model}"}
+        options = {:model => "ManageIQ::Providers::ConfigurationManager::InventoryGroup", :match_via_descendants => ConfiguredSystem, :where_clause => ["ems_id IN (?)", provider.id]}
+        @no_checkboxes = true
+        process_show_list(options)
+        record_model = ui_lookup(:model => model_to_name(model ? model : model_to_name(TreeBuilder.get_model_for_prefix(@nodetype))))
```

Comment:
I think this is meant to be the same as https://github.com/ManageIQ/manageiq/pull/8134/files#diff-dd586a3167b910b76243edbecbbf0de5R539